pkgs/tools/misc/ocs-url/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchgit, libsForQt5, git, lib }:
+
+stdenv.mkDerivation rec {
+  name = "ocs-url";
+  version = "3.1.0";
+
+  meta.description = "Open Collaboration System for use with DE store websites";
+  meta.license = lib.licenses.gpl3Only;
+  meta.maintainers = [ lib.maintainers.SohamG ];
+  meta.platforms = lib.platforms.linux;
+  srcs = [
+    (fetchgit {
+      url = "https://www.opencode.net/dfn2/ocs-url.git";
+      rev = "refs/tags/release-3.1.0";
+      sha256 = "RvbkcSj8iUAHAEOyETwfH+3XnCCY/p8XM8LgVrZxrws=";
+    })
+
+    (fetchgit {
+      url = "https://github.com/akiraohgaki/qtil";
+      rev = "refs/tags/v0.4.0";
+      sha256 = "XRSp0F7ggfkof1RNAnQU3+O9DcXDy81VR7NakITOXrw=";
+    })
+  ];
+
+  sourceRoot = "ocs-url";
+  dontWrapQtApps = true;
+
+  buildInputs = with libsForQt5.qt5;
+    [ qtbase qtsvg qtquickcontrols qmake ] ++ [ git ];
+
+  buildPhase = ''
+    source $stdenv/setup
+    runHook preBuild
```

Comment:
I think you could make `buildPhase` a default if you moved things around:

1. Move to `postPatch` (or `postUnpack`) source fiddling:
```
    mkdir -p ./lib/qtil
    cp -r ../qtil/* lib/qtil/
```

2. Move to `configurePhase` (or `postConfigure`): `qmake PREFIX=$out`.

As a benefit you would get `make` ran with parallelism enabled by default.